libs/ui/src/app_base.tsx

```diff
@@ -50,6 +53,13 @@ export function AppBase(props: AppBaseProps): JSX.Element {
   const [colorMode, setColorMode] = React.useState<ColorMode>(defaultColorMode);
   const [sizeMode, setSizeMode] = React.useState<SizeMode>(defaultSizeMode);
 
+  React.useEffect(() => {
+    /* istanbul ignore next - tested via integration tests. */
+    if (!disableFontsForTests) {
+      loadFonts();
+    }
+  }, [disableFontsForTests]);
```

Comment:
Does this mean there will be an initial render without fonts loaded? Does that cause any visual weirdness?

Reply:
Yeah, good question - I didn't see any noticeable flicker on initial render when I was testing, but I'm going to try it out with caching disabled to see what kind of impact it might have

Reply:
Couldn't spot it while watching the refresh, but managed to catch a frame on a screen recording where the text is in sans-serif just before switching to Roboto. Just to be safe, I'm going to delay the rendering until `loadFonts` has run

Reply:
Nice, thanks for looking into that